integrationTests/package-lock.json

```diff
@@ -0,0 +1,9560 @@
+{
```

Comment:
@monkey-jeff @bredmond-sf Should this get added to .gitignore? The others don't seem to be, but they're also generated...

Reply:
Have you tried npm pack with package-lock already present? It causes integrity errors for the new pack files during npm install step